From 06a9f8e3ab72fb0c8eab35b7d70c75f9939a3ac7 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 9 Jul 2018 15:32:28 +0800
Subject: [PATCH] Revert "arm64: defconfig: rk3308_linux_defconfig: enable
 CONFIG_CYW_BCMDHD"

This reverts commit e2e7ae22030347c2593c5990ef998642518c94dd.
This is a temporary workaround.

Change-Id: I77dbfec1e5e4d9921bc8b8bf6fc8687e7b1744e2
Signed-off-by: Xiao Yao <xiaoyao@rock-chips.com>
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rk3308_linux_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rk3308_linux_defconfig b/arch/arm64/configs/rk3308_linux_defconfig
index a1555cb1479f..7702eb755c81 100644
--- a/arch/arm64/configs/rk3308_linux_defconfig
+++ b/arch/arm64/configs/rk3308_linux_defconfig
@@ -95,7 +95,7 @@ CONFIG_NETDEVICES=y
 # CONFIG_ETHERNET is not set
 CONFIG_WL_ROCKCHIP=y
 CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
-CONFIG_CYW_BCMDHD=y
+CONFIG_AP6XXX=y
 # CONFIG_RTL_WIRELESS_SOLUTION is not set
 # CONFIG_INPUT_MOUSEDEV is not set
 CONFIG_INPUT_EVDEV=y
-- 
2.35.3

